Dockerfile

```diff
@@ -1,9 +1,17 @@
+# temporary image: build an xsnippet-api wheel and collect all its dependencies
```

Comment:
nit: I don't think these comments make any sense as it's clearly stated in Docker docs that the last image in your Dockerfile is what actually will be published.

Reply:
ok, I'll remove them!